webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require("path");
+const merge = require("webpack-merge");
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+const common = {
+  entry: "./src/index",
+  output: {
+    library: "Kinto",
+    libraryTarget: "umd",
+    filename: "kinto.js",
+    path: path.resolve(__dirname, "dist")
+  },
+  module: {
+    loaders: [{
+      test: /\.js$/,
+      exclude: /node_modules/,
+      loader: "babel-loader"
+    }]
+  },
+  devtool: "source-map"
+};
+
+switch (process.env.npm_lifecycle_event) {
+case "dist-prod":
```

Comment:
I only did that because of your lint rules. If I make 4 spaces my editor complains about the linting rules.